### PR TITLE
Store analytics Component

### PR DESCRIPTION
**Pull request by prasanthreddy22**

### CODE REVIEW

File: src/components/StoreAnalytics.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { __ } from "@wordpress/i18n";
+export default function StoreAnalytics(props) {
```
Comment (aulisius) [inline src/components/StoreAnalytics.js:2]: Let's do a named export

File: src/components/StoreAnalytics.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                </h2>
+                <div style={{height: "8px"}}/>
+                <span className="store-analytics-notice">
+                    {__(
+                        "Get a detailed view of all your store performance analytics via the WooCommerce Analytics page.",
+                        "wp-module-ecommerce"
+                    )}
+                </span>
+                <div style={{height: "16px"}}/>
+                <a href={"admin.php?page=wc-admin&path=/analytics/overview&return_to_nfd=/home/store/"+props.section}>
```
Comment (aulisius) [inline src/components/StoreAnalytics.js:23]: Use `URLSearchParams`

File: styles.scss [hunk 3]
```diff
@@ -316,4 +316,30 @@
...
+.nfd-ecommerce-store-analytics {
+  border: 1px solid #D7DBE0;
+  background-color: white;
+  padding-left: 24px;
+  padding-top: 24px;
+  padding-bottom: 19px;
+
+  .store-analytics-notice {
+    font-size : 16px ;
+    color: #425466;
```
Comment (aulisius) [inline styles.scss:330]: Use CSS variable

**Review by aulisius — CHANGES_REQUESTED** [review]

File: src/components/StoreAnalytics.js (hunk 1 above)
Comment (prasanthreddy22) [reply src/components/StoreAnalytics.js]: done

File: src/components/StoreAnalytics.js (hunk 2 above)
Comment (prasanthreddy22) [reply src/components/StoreAnalytics.js]: done

File: styles.scss (hunk 3 above)
Comment (prasanthreddy22) [reply styles.scss]: done

**Pull request merged**